package.json

```diff
@@ -29,8 +29,12 @@
     "fix": "xo --fix",
     "mocha": "mocha \"test/**/*.test.js\"",
     "coverage": "c8 npm run mocha",
+    "coverage:jest": "c8 npm run test:jest",
```

Comment:
let's just keep jest's default coverage and remove c8 dev dependency

Reply:
Ok but it is still in draft :D

Reply:
I think this should be done in another PR right after this one is merged. Otherwise this PR will be HUGE. But if it is ok I will change it after tests review

Reply:
Well, as this PR is right now, it's incomplete. It's not being used on CI and it's not final. I'd say better make all the changes related to the switch to jest here and remove everything that's not needed, but keep the old npm scripts.

Reply:
OK